src/tbb/tools_api/disable_warnings.h

```diff
@@ -1,29 +1,23 @@
 /*
-    Copyright (c) 2005-2021 Intel Corporation
+  Copyright (C) 2005-2019 Intel Corporation
 
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
+  SPDX-License-Identifier: GPL-2.0-only OR BSD-3-Clause
```

Comment:
Also, the license should be remained as-is that is Apache because we use ittnotify source code, modify it if necessary and license under Apache.